src/components/App/App.js

```diff
@@ -44,29 +208,61 @@ function App() {
   }, []);
 
   return (
-    <>
-      <Header
-        handleScreenResize={handleScreenSize}
-        onSignInClick={handleOpenModal}
-        screenWidth={screenWidth}
-        isLoggedIn={isLoggedIn}
-        modalIsOpen={modalIsOpen}
-        isMobileMenuOpen={showMobileMenu}
-        handleOpenMenu={handleToggleMenu}
-      />
-      <Switch>
-        <Route exact path='/'>
-          <Main isLoggedIn={isLoggedIn} />
-        </Route>
-        <Route path='/saved-news'>
-          <Main isLoggedIn={isLoggedIn} />
-        </Route>
-      </Switch>
-      <Footer />
-      <Modal isOpen={modalIsOpen} onClose={handleCloseModal}>
-        <Form onClose={handleCloseModal} />
-      </Modal>
-    </>
+    <CurrentUserContext.Provider value={currentUser}>
+      <>
```

Comment:
` <>` empty tags are not needed if you have a common tag that wraps all the markup of the component